apps/GNOME/uninstall

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+purge_packages gnome gnome-core gnome-software gnome-shell || error 1
```

Comment:
purge_packages is all you call. it does not take any input

also error 1 is incorrect. that will write `1` to the terminal and exit

Reply:
Ah, okay, that makes sense. If I'm understanding correctly,  `error` outputs an error message and exits with exit code `1` whilst `exit` exits with the specified error code - correct?

Reply:
yes